### DISCUSSION TITLE
Enable search in editor with select make

### DISCUSSION THREAD

**Discussion by markplus** [yajra/laravel-datatables#3079] [Q&A]
I created a column where when I create a new article I have a selection menu with the select in this way, but I also wanted to have the possibility of searching soas not to scroll through the list but how can I integrate it?

when I click again in the editor I have the list of names but I can't search for a name I have to scroll through the list, how do I enable the search?

```
return $this->builder()
                    ->setTableId('users-table')
                    ->columns($this->getColumns())
                    ->minifiedAjax()
                    ->orderBy(0)
                    ->selectStyleSingle()
                    ->editors([
                        Editor::make()
                        ->fields([
                            Fields\Text::make('descrizione'),
                            Select::make('users')->options(['','name',name2','name3','name4']),
```

**Comment by yajra** [comment]
You can use the Select2 field plugin of the Editor and possibly do a server-side search with it.

**Reply by markplus** [reply]
I tried to insert Select2 but the table disappears, Select::make works but I don't have the search field inside the selection, if I insert Select2::make the table disappears from the view page
